packages/extension/src/routes/Step2.vue

```diff
@@ -32,10 +32,17 @@
                  :aria-disabled="disableButton" :class="{disabled: disableButton}">
       I'm all set
     </router-link>
+    <a
```

Comment:
I believe a `button` element is more appropriate for this, as this is not really a link